### PR TITLE
Fix integration tests and add cleanup workflow 

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added or updated automated unit tests to verify correctness of my new code.
   * [ ] I've added or updated integration tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually on MacOS.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually on Windows.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

**Review by sfc-gh-pjob — DISMISSED** [review]

File: .github/workflows/cleanup.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          persist-credentials: false
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip hatch
+          python -m hatch env create e2e
+      - name: Run end to end tests
```
Comment (sfc-gh-astus) [inline .github/workflows/cleanup.yaml:23]: Run cleaunp

File: scripts/cleanup.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        print("ERROR", msg)
+        print(str(err)[:200])
+
+
+def remove_resources(single: str, plural: str, known_instances: t.List[str], role: str):
+    items = session.sql(f"show {plural}").collect()
+
+    for item in items:
+        if item.name not in known_instances:
+            resource_delete(resource_type=single, item=item, role=role)
```
Comment (sfc-gh-astus) [inline scripts/cleanup.py:48]: Maybe `remove_resource`?

**Comment by sfc-gh-astus** [comment]
PR title says only part of things added in PR :)

**Review by sfc-gh-astus — APPROVED** [review]

File: .github/workflows/cleanup.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+name: Environment cleanup
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 10 * * *"
```
[the remark this answers is not in the document]
Comment (sfc-gh-turbaszek) [reply .github/workflows/cleanup.yaml:6]: @sfc-gh-pjob the script removes only items created 2 ours before run so I think we should be fine.

[the remark this answers is not in the document]
Comment (sfc-gh-pjob) [reply .github/workflows/cleanup.yaml:6]: yes, you're right - I forgot about it

**Review by sfc-gh-pjob — APPROVED** [review]

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**